variables.tf

```diff
@@ -172,3 +172,54 @@ variable "tags" {
   description = "(optional) Tags for VPC resources"
   default     = {}
 }
+
+variable "enable_vpc_flow_log_to_cloudwatch" {
+  type        = bool
+  description = "Flag to enable or disable VPC flow logs to Cloudwatch."
+  default     = false
+}
+
+variable "deletion_window_in_days" {
+  type        = number
+  default     = 10
+  description = "Duration in days after which the key is deleted after destruction of the resource"
+}
+
+variable "enable_key_rotation" {
+  type        = bool
+  default     = true
+  description = "Specifies whether key rotation is enabled"
+}
```

Comment:
KMS related variable has to be one variable using object

Reply:
corrections made